scripts/linters/pylint_extensions.py

```diff
@@ -590,14 +590,14 @@ class DocstringParameterChecker(checkers.BaseChecker):
     DOCSTRING_SECTION_RAISES = 'raises'
 
     def visit_classdef(self, node):
-        """Visit each class definition in a module and check if there is a
-        single new line below each class docstring.
+        """Visit each class definition and check if there is a
+        period at the end for each class docstring.
 
         Args:
-            node: astroid.nodes.ClassDef. Node for a class definition
+            node: astroid.nodes.Classdef. Node for a class definition
```

Comment:
Please revert this change. The earlier name was correct.

Reply:
Yeah sure!